great_expectations/render/renderer/site_builder.py

```diff
@@ -291,16 +325,22 @@ def build(self, resource_identifiers=None):
                     logger.debug("        Rendering profiling for batch {}".format(resource_key.batch_identifier))
                 else:
 
-                    logger.debug("        Rendering validation: run id: {}, suite {} for batch {}".format(run_id,
-                                                                                                              expectation_suite_name,
-                                                                                                              resource_key.batch_identifier))
+                    logger.debug(
+                        "        Rendering validation: run id: {}, suite {} for batch {}".format(
+                            run_id,
+                            expectation_suite_name,
+                            resource_key.batch_identifier
+                        )
+                    )
 
             try:
                 rendered_content = self.renderer_class.render(resource)
                 viewable_content = self.view_class.render(rendered_content)
             except Exception as e:
-                logger.error("Exception occurred during data docs rendering: ", e, exc_info=True)
-                continue
+                logger.error('Exception occurred during data docs rendering: ', e, exc_info=True)
+                exception_traceback = traceback.format_exc()
+                exception_message = f'{type(e).__name__}: "{str(e)}".  Traceback: "{exception_traceback}".'
+                raise exceptions.GreatExpectationsError(exception_message)
```

Comment:
Again. I'm uncertain about this re-raise.

Reply:
Same motivation as above. Happy to discuss.